### DISCUSSION TITLE
support output to asciicast format in standard script(1) / scriptreplay(1)

### DISCUSSION THREAD

**Discussion by karelzak** [asciinema/discussions#84] [Ideas]
I'd like to make it simple for people who use script(1) to publish their typescripts. The best would be to add --convert=asciicast to scriptreplay(1), so users with old typescripts from the script(1) can use asciinema tools. I think it's bad that standard Linux tools are out of the game :-) 

What do you think about this idea?

Frankly, I have no clue how complicated is asciicast, but I guess the data part of the JSON is just raw terminal typescript with some encoding for compatibility with JSON, right?

**Comment by ku1ik** [comment]
asciinema player v3.4.0 added support for typescript and ttyrec recordings: https://github.com/asciinema/asciinema-player/blob/develop/src/parser/README.md#typescript

At the moment this is only useful for people who host the player themselves on their own sites, as asciinema.org doesn't support upload of those file types, but still an option for those who can self-host the player.

**Comment by FranklinYu** [comment]
Possible duplicate of #79?
